.editorconfig

```diff
@@ -0,0 +1,16 @@
+# editorconfig.org
```

Comment:
Do we need this file to be committed?

Reply:
What's the difference between force eslint rules and use a editorconfig file to keep this settings on editors?

Both of them helps with the same issue, provide a ruleset for code formatting

Reply:
+1 This file is useful to keep consistency in editors w/o complex plugins

Reply:
If you change some rules in eslintrc but you don't do the same in editorconfig, what happens?